examples/official-storybook/stories/addon-docs/addon-docs.stories.mdx

```diff
@@ -28,6 +28,10 @@ import DocgenButton from '../../components/DocgenButton';
 
 <Props of="." />
 
+<Preview>
+  <Story id="empty-story" />
```

Comment:
Should this be `id="nonexistent-story"`? The `id` attribute is to refer to an existing story that's been defined somewhere else in storybook.

Reply:
thanks, changed it. I guess I have been looking at empty stories too much :)